op-e2e/faultproof_test.go

```diff
@@ -558,8 +558,10 @@ func setupDisputeGameForInvalidOutputRoot(t *testing.T, outputRoot common.Hash)
 }
 
 func TestCannonChallengeWithCorrectRoot(t *testing.T) {
-	InitParallel(t, UsesCannon)
-
+	InitParallel(t, UsesCannon, UseExecutor(0))
+	if true {
+		return
```

Comment:
Is this intended?

Reply:
Yeah that's definitely a bad idea.  Thanks for flagging.

Reply:
Fixed.